src/components/Table/TableRow.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import React from 'react';
+import theme from '../../theme';
+import { COLOR, COMMON, TYPOGRAPHY } from '../../utils/constants';
+import TableHeadContext from './TableHeadContext';
+
+
+export const TableRowWrapper = styled.tr`
+    height: 52px;
+    ${(props) => props.isHeadRow && css`
+        height: 56px;
+    `}
+    ${COLOR};
+    ${COMMON};
+    ${TYPOGRAPHY};
+`;
+
+const TableRow = ({ children, ...props }) => {
+  const tableHeadContext = React.useContext(TableHeadContext);
+  const isHeadRow = tableHeadContext && tableHeadContext.isHead === true;
```

Comment:
probably don't need the `=== true`

if you want to make sure isHeadRow is a boolean (in the case that tableHeadContext.isHead is undefined)

you could do 
```Javascript
const isHeadRow = !!(tableHeadContext && tableHeadContext.isHead);
```

but that probably isn't necessary